ui/src/app/app-routing.tsx

```diff
@@ -98,6 +100,13 @@ export const AppRoutingComponent: React.FunctionComponent<RoutingProps> = ({onSi
           path='/admin/workspace-audit/:workspaceNamespace'
           component={() => <WorkspaceAuditPage routeData={{title: 'Workspace Audit'}}/>}
       />
+      <AppRoute
```

Comment:
I think you can add a friendly `title` property to the `routeData` to give yourself a tab title.

Reply:
`pathElementForTitle: 'nbName'` does this: the notebook name is the tab title.

Reply:
Maybe `Admin Inspection View for $nbName` or something would give a bit more context?